backend/app/api/routes/inquiries.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+
+from fastapi import APIRouter, HTTPException
+
+from app import crud
+from app.api.deps import SessionDep
+from app.models import Inquiry, InquiryCreate, InquiryPublic
+
+router = APIRouter()
+
+
+@router.post("/", response_model=InquiryPublic)
+def create_inquiry(
+    *, session: SessionDep, inquiry_in: InquiryCreate
+) -> Any:
```

Comment:
just curious -- is Any return type a good practice in Python? In general, I would recommend strong typing whenever possible.

Reply:
Sorry I'm confused. In that page they're having it return a typed value. For example, 
<img width="379" alt="image" src="https://github.com/user-attachments/assets/488ebb26-f87a-40fa-801e-13b09080b17f">

Reply:
Kind of looks like the any type is used in this case:

```
There are some cases where you need or want to return some data that is not exactly what the type declares.

For example, you could want to return a dictionary or a database object, but declare it as a Pydantic model. This way the Pydantic model would do all the data documentation, validation, etc. for the object that you returned (e.g. a dictionary or database object).
```

Is this such a case?

Reply:
Read further down the page
![fapi1](https://github.com/user-attachments/assets/b99a2e2a-ebea-4dd0-a36e-f84af5979da9)

Reply:
In any case, I have changed the return type in accord with your comment.